dev/Dockerfile

```diff
@@ -129,7 +129,7 @@ RUN ./configure
 FROM build_dependencies AS build
 
 ARG checkout_branch=master
-ARG configure_options
+ARG configure_options="-c --spidermonkey-version 60"
```

Comment:
This is a little surprising because it gets dropped if a user specifies `configure_options` on the command line. I got bitten by it when my usual `--build-arg configure_options="--disable-docs"` invocation stopped working after this change.

Also, why specify `-c` here?

Reply:
@kocolosk Yeah, I'm not sure how to handle this more elegantly; feel free to propose a solution. The default choice of `1.8.5` doesn't work in the image now that we're selectively only installing libmozjs-60, which was a conscious decision so I can gradually deprecate the 1.8.5 packages. But we can't make the new default 60 yet, since it will fail on just about every other OS at the moment.

Without `-c` you can't use the libcurl bindings in javascript, which developers arguably might want to do.